src/viam_mlmodelservice_triton_impl.cpp

```diff
@@ -103,6 +103,12 @@ class Service : public vsdk::MLModelService, public vsdk::Stoppable, public vsdk
             std::lock_guard<std::mutex> lock(state_lock_);
             if (!stopped_) {
                 stopped_ = true;
+
+                // Remove any symlinks we've added to our module workspace.
```

Comment:
What results on restart if the triton module crashes or is killed and this removal doesn't happen?

Reply:
If you're using the same model name on restart, everything is fine (we delete the symlink and make a new one which might be identical to the old one). If you're using a different model, the Triton server will load both the new one and the old no-longer-used one, and waste GPU memory.